packages/mendel-pipeline/src/registry/client.js

```diff
@@ -110,7 +104,11 @@ class MendelOutletRegistry {
             return positives.some(g => g.match(id)) &&
                 negatives.every(g => g.match(id));
         })
-        .map(id => this.getEntry(id));
+        .map(id => this.getEntry(id))
+        .filter(({variation}) => {
+            return !excludeVariational ||
+                variation === this._options.baseConfig.dir;
+        });
```

Comment:
Why do we exclude variational in here? For instance, what about a variation that introduces a bundle? The _entrance_ would only exist in the new bundle. I was under the impression that picking up from all variation dirs was the proper behavior. It might be breaking something, but Not in full example, at least.